### ISSUE TITLE
thinkpad module: Refusing to run without usable config file!

### ISSUE THREAD

**Issue by coretemp** [NixOS/nixpkgs#40452]
## Issue description

Specifying 
```
services.thinkfan.enable = true;
```
results in 

```
Refusing to run without usable config file!
```

Even this fails:
```
services.thinkfan.enable = true;
services.thinkfan.sensor = "/proc/acpi/ibm/fan";
```

### Steps to reproduce
The file I do have is:

```
cat /proc/acpi/ibm/fan 
status:         enabled
speed:          3986
level:          auto
```

 - system: `"x86_64-linux"`
 - host os: `Linux 4.15.18, NixOS, 18.03.git.ef74caf (Impala)`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.0.1`
 - channels(root): `"nixos-17.09.2182.7f6f0c49f0"`
 - channels(coretemp): `"nixos-18.03.132192.ce0d9d638de"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos/nixpkgs`

**Comment by lheckemann** [comment]
Fixed by #40768 AFAIU.

**Comment by truthadjustr** [comment]
I have a Thinkpad T440. This is how I successfully configured it: https://gist.github.com/truthadjustr/b48c352c2d44e9529b8e427a9e500337
